Cargo.toml

```
@@ -526,4 +526,4 @@ rev = "6105d7a5591aefa646a95d12b5e8d3f55a9214ef"
 #
 [patch.crates-io.curve25519-dalek]
 git = "https://github.com/solana-labs/curve25519-dalek.git"
-rev = "c14774464c4d38de553c6ef2f48a10982c1b4801"
+branch = "3.2.1-unpin-zeroize"
```

My bad.  I pushed commits from my personal fork into `solana-labs/curve25519-dalek.git` and thought that the commit hashes would be the same.
I could swear I thought I have checked it as well.

I think it is better to use an exact hash, rather than a branch, as we specifically  want this particular commit:
```suggestion
branch = "b500cdc2a920cd5bff9e2dd974d7b97349d61464"
```

We also need to change it in the comment above.
I can send a PR in a minute...